etc/omero.properties

```diff
@@ -67,8 +67,8 @@ omero.db.prepared_statement_cache_size=10
 # you can modify the defaults.
 #
 # Note: some other properties are defined in
-# the file etc/profiles/${omero.db.profile}
-# Especially of importance is omero.db.port
+# the file :file:`etc/profiles/${omero.db.profile}`
```

Comment:
Understood, this means Sphinx must escape those then. Should I remove the `file` markup and default to regular verbatim?